tools/clang/unittests/HLSL/ShaderOpTest.cpp

```diff
@@ -717,7 +717,9 @@ void ShaderOpTest::CreateShaders() {
       CHECK_HR(pResult->GetResult(&pCode));
       CComPtr<IDxcBlobEncoding> pBlob;
       CHECK_HR(pCompiler->Disassemble((IDxcBlob *)pCode, (IDxcBlobEncoding **)&pBlob));
-      dxc::WriteUtf8ToConsoleSizeT((char *)pBlob->GetBufferPointer(), pBlob->GetBufferSize());
+      CComPtr<IDxcBlobEncoding> pUtf16Blob;
+      pLibrary->GetBlobAsUtf16(pBlob, &pUtf16Blob);
+      hlsl_test::LogCommentFmt(L"%*s", (int)pUtf16Blob->GetBufferSize() / 2, (LPCWSTR)pUtf16Blob->GetBufferPointer());
```

Comment:
/ sizeof(wchar_t). This assumes that the buffer has no padding, and will probably log the null terminator if there's one, but that's probably no big deal.

Reply:
This is just for debugging purposes, under #if 0 in shipping code.